setup.py

```diff
@@ -186,7 +186,7 @@ def run(self):
 
 setup(
     name="diffusers",
-    version="0.2.4",
+    version="0.2.4.dev0", # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
```

Comment:
```suggestion
    version="0.3.0.dev0", # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
```
-> Let's do Transformers style here then as well

Reply:
cc @LysandreJik @sgugger

Reply:
indeed, bumped up the version 66f5d1e4cda6b69f7e15921457a77ce072873e1d